sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -53,6 +53,8 @@ private Sentry() {}
    *
    * <p>For Android options will also be (temporarily) replaced by SentryAndroid static block.
    */
+  // TODO [HSM] use SentryOptions.empty and address
```

Comment:
When simply using empty options, should we even set something different statically for Android? Should there also be a `SentryAndroidOptions.empty()`?